From: Ben Collins <bcollins@ubuntu.com>
Date: Tue, 18 Oct 2005 20:12:36 +0000 (-0400)
Subject: [UBUNTU:scsi/BusLogic] Add MODULE_DEVICE_TABLE
X-Git-Url: http://kernel.org/git/?p=linux/kernel/git/bcollins/ubuntu-2.6.git;a=commitdiff;h=94fea61c7b73fdcaa3ff349bd201f6690802c2f2

[UBUNTU:scsi/BusLogic] Add MODULE_DEVICE_TABLE

PatchAuthor: Ben Collins <bcollins@ubuntu.com>
UpstreamStatus: Unsubmitted

Signed-off-by: Ben Collins <bcollins@ubuntu.com>
---

--- a/drivers/scsi/BusLogic.c
+++ b/drivers/scsi/BusLogic.c
@@ -3569,5 +3569,18 @@ static void __exit BusLogic_exit(void)
 
 __setup("BusLogic=", BusLogic_Setup);
 
+
+/* PCI Devices supported by this driver */
+static struct pci_device_id BusLogic_pci_tbl[] __devinitdata = {
+	{ PCI_VENDOR_ID_BUSLOGIC, PCI_DEVICE_ID_BUSLOGIC_MULTIMASTER,
+	  PCI_ANY_ID, PCI_ANY_ID, 0, 0, 0},
+	{ PCI_VENDOR_ID_BUSLOGIC, PCI_DEVICE_ID_BUSLOGIC_MULTIMASTER_NC,
+	  PCI_ANY_ID, PCI_ANY_ID, 0, 0, 0},
+	{ PCI_VENDOR_ID_BUSLOGIC, PCI_DEVICE_ID_BUSLOGIC_FLASHPOINT,
+	  PCI_ANY_ID, PCI_ANY_ID, 0, 0, 0},
+	{ }
+};
+MODULE_DEVICE_TABLE(pci, BusLogic_pci_tbl);
+
 module_init(BusLogic_init);
 module_exit(BusLogic_exit);
